frontend/src/components/pages/your_clubs/YourClubs.tsx

```diff
@@ -1,10 +1,287 @@
-import * as React from 'react';
+import * as React from "react";
+import AddIcon from "@material-ui/icons/Add";
+import { BookProps } from
+  "../../../services/mock_backend/mock_your_clubs_backend";
+import { borders } from "@material-ui/system";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import { ClubProps } from
+  "../../../services/mock_backend/mock_your_clubs_backend";
+import { createStyles } from "@material-ui/core/styles";
+import { defaultServices } from "../../contexts/contexts";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import HighlightOffIcon from "@material-ui/icons/HighlightOff";
+import InputLabel from "@material-ui/core/InputLabel";
+import LibraryBooksRoundedIcon from "@material-ui/icons/LibraryBooksRounded";
+import { makeStyles } from "@material-ui/core/styles";
+import MenuItem from "@material-ui/core/MenuItem";
+import PageviewIcon from "@material-ui/icons/Pageview";
+import Select from "@material-ui/core/Select";
+import { ServiceContext } from "../../contexts/contexts";
+import { Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+      flexDirection: 'column',
+      flexWrap: 'wrap',
+      marginTop: '100px',
+      marginLeft: theme.spacing(4),
+    },
+    break : {
+      flexBasis: '100%',
+      height: 0,
+    },
+    button : {
+      margin: theme.spacing(1),
+      marginLeft: theme.spacing(0),
+      marginTop: '20px',
+      maxHeight: '50px',
+    },
+    buttonsContainer : {
+      alignItems: 'right',
+      display: 'flex',
+    },
+    club : {
+      marginTop: '20px',
+      maxWidth: '900px',
+    },
+    clubContent: {
+      display: 'flex',
+      flexWrap: 'wrap',
+      margin: theme.spacing(1),
+    },
+    clubPhoto: {
+      marginTop: '15px',
+    },
+    clubTitle: {
+      marginBottom: 0,
+      marginRight: 50, 
+    },
+    formControl: {
+      margin: theme.spacing(1),
+      marginBottom: '5px',
+      marginRight: theme.spacing(8),
+      minWidth: 120,
+    },
+    listedClubsContainer: {
+      alignItems: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: '20px',
+    },
+    textElement: {
+      marginBottom: '0px',
+    },
+    topUtilitiesContainer: {
+      display: 'flex',
+      justifyContent: 'center',
+    },
+  }),
+);
 
 export const YourClubs = () => {
+  const classes = useStyles();
+
+  /**
+   * ServiceHandlers is an object containing various TS Handlers and provides
+   * functionality to communicate data from the frontend to the backend
```

Comment:
```suggestion
   * functionality to communicate data from the frontend to the backend.
```
Period at end of comment.